src/app/workflow/info-tab/info-tab.component.html

```diff
@@ -54,6 +54,16 @@
         <li *ngIf="isPublic && isValidVersion">
           <strong matTooltip="TRS link to the main descriptor for the selected workflow version">TRS</strong>:
           <a [href]="trsLink"> #{{ entryType$ | async }}/{{ workflow?.full_workflow_path }} </a>
+          <button
+            mat-icon-button
+            color="secondary"
+            ngxClipboard
+            displayText="#{{ entryType$ | async }}/{{ workflow?.full_workflow_path }}"
+            [cbContent]="displayText"
```

Comment:
There is no displayText property; this doesn't work for me when I run it and click on the button.